naming/src/main/java/com/alibaba/nacos/naming/remote/rpc/handler/InstanceRequestHandler.java

```diff
@@ -51,6 +52,7 @@ public InstanceRequestHandler(EphemeralClientOperationServiceImpl clientOperatio
     public InstanceResponse handle(InstanceRequest request, RequestMeta meta) throws NacosException {
         Service service = Service
                 .newService(request.getNamespace(), request.getGroupName(), request.getServiceName(), true);
+        InstanceUtil.setInstanceIdIfEmpty(request.getInstance(), request.getServiceName());
```

Comment:
Is this will lose the group name?

Reply:
> 

Thank you for your reminder, I have already fixed this issue, please review it again.